Formula/tracker.rb

```diff
@@ -48,12 +58,9 @@ def install
     ]
 
     ENV["DESTDIR"] = "/"
-    mkdir "build" do
-      system "meson", *args, ".."
-      # Disable parallel build due to error: 'libtracker-sparql/tracker-sparql-enum-types.h' file not found
-      system "ninja", "-v", "-j1"
-      system "ninja", "install", "-v"
-    end
+    system "meson", ".", "build", *args, *std_meson_args
```

Comment:
Should use `meson setup` since default/implied `setup` is deprecated.

Ref: https://mesonbuild.com/Commands.html#setup

Reply:
👍